### DISCUSSION TITLE
Admin of collection with ability to manage collection, but don't include in results (hidden collection user setting?)

### DISCUSSION THREAD

**Discussion by nneul** [dani-garcia/vaultwarden#2312] [Ideas]
Not sure of the best way to describe/do this - but the idea is to have a privilege level that would allow for having a collection "hidden", yet still have permissions. Example would be a security lead or similar - someone with access to a whole bunch of collections for an org, but you don't necessarily want all of those sub-collections showing up in the search results/auto fill/etc. 

A workaround would be "just use an alternate account for the 'administration' function" but that's not always suitable.

**Comment by jjlin** [comment]
You can do this by selecting `This user can access only the selected collections` for the admin, and then deselecting the collections to be hidden. Those collections will be hidden in the admin's personal vault view, but will still be accessible from the org vault view.
